answerkey/testing/11.answer.scala

```diff
@@ -2,11 +2,16 @@ case class SGen[+A](g: Int => Gen[A]) {
   def apply(n: Int): Gen[A] = g(n)
 
   def map[B](f: A => B): SGen[B] =
-    SGen(g andThen (_ map f))
+    SGen { g(_) map f }
+  //SGen(g andThen (_ map f))
```

Comment:
If you leave this comment here, please have it explain why it's here (e.g. "alternate implementation"). Otherwise, just remove it.